examples/loopback.rs

```diff
@@ -164,13 +167,15 @@ fn main() {
         }
 
         match iface.poll_delay(&socket_set, clock.elapsed()) {
-            Some(0) =>
```

Comment:
now that `millis` is `pub`, I think you can match directly

Reply:
Good point.